app/views/alchemy/admin/pictures/_archive.html.erb

```diff
@@ -1,5 +1,5 @@
 <div id="library_sidebar">
-  <%= render 'filter_bar' %>
+  <%= render 'alchemy/admin/resources/filter_wrapper' if resource_has_filters %>
```

Comment:
All other `render` calls use the partial name w/o the path. Why was this necessary here? Rails should be able to resolve the path. (And the same comment with the renamed `filter_bar` applies here as well)

Reply:
I remember stumbling over this and it felt awkward when it failed. I didn't dig deeper and just changed the path to make it work. I'll check again.

Reply:
No clue why it didn't work when I was working on it a few weeks ago. Now it works just fine without the path...